apps/web-staking/src/assets/abi/esXaiAbi.ts

```diff
@@ -552,7 +552,7 @@ export const esXaiAbi = [
 						"type": "bool"
 					}
 				],
-				"internalType": "struct esXai3.RedemptionRequestExt",
+				"internalType": "struct esXai6.RedemptionRequestExt",
```

Comment:
Should this be esXai3 for mainnet?

Reply:
Good catch. The name for the struct does not matter as far as I know, however, we should leave it as what it actually is for consistency